src/Employer/Employer.Web/wwwroot/lib/accessible-autocomplete/accessible-autocomplete.min.js

```diff
@@ -0,0 +1,805 @@
+! function(e, t) {
```

Comment:
In provider there is a searchSuggestionService.js which does similar stuff. May be reuse or clone it in employers.

Reply:
Yes. We should consolidate these at some point. The searchSuggestionService uses jQuery UI's auto-complete. But I've been explicitly told that this should use GDS accessible-autocomplete. We also use Select2 in places. I've had a chat to Steve H about this and he's aware of this situation.